### PR TITLE
CIVDT-1359: Additional changes

### PR DESCRIPTION
**Pull request by jacemale-hmrc**
Main changes:
- extracted common file upload messages
- replaced heading/pageHeader with pageTitle wherever possible
- removed code that used to deal with titles and headers separately
- removed header messages from test message files

### CODE REVIEW

File: test/assets/messages/cancelCase/CancelCaseUploadSupportingDocumentationMessages.scala [hunk 1]
```diff
@@ -16,26 +16,10 @@
...
-  val button: String       = "Upload chosen file"
-  val fileUploadId: String = "file"
-  val fileTooSmall         = "Select a file to upload"
-  val fileTooBig           = "The selected file must be smaller than 6MB"
-  val fileUnknown          = "The selected file could not be uploaded – try again"
-  val fileRejected =
-    "The selected file must be a PDF, XLS, XLSX, DOC, DOCX, PPT, PPTX, ODT, ODS, ODP, JPG, PNG, or TIFF"
-  val fileQuarantined = "The selected file contains a virus"
+  val title: String = "Upload documentation"
+  val h1: String    = "Upload documentation"
```
Comment (jakebooker56) [inline test/assets/messages/cancelCase/CancelCaseUploadSupportingDocumentationMessages.scala:24]: do we need this duplication or can we just have title and remove h1?

File: conf/messages.cy [hunk 2]
```diff
@@ -357,6 +357,17 @@ enterCustomsProcedureCode.hint = This can be 7 numbers, or 6 numbers and a lette
...
+common.uploadFile.PDF = PDF (.pdf)
+common.uploadFile.CSV = CSV (.csv)
+common.uploadFile.ms = Microsoft Excel, Word or PowerPoint (.xls, .xlsx, .doc, .docx, .ppt or .pptx)
+common.uploadFile.openDocumentFormat = Open Document Format (.odt, .ods or .odp)
+common.uploadFile.image = image (.jpeg, .jpg, .png or .tiff)
+common.uploadFile.fileSize = Each file must be 6MB or less and you can only upload one file at a time.
+
 # Upload File page
 uploadFile.pageTitle = Upload supporting documentation
 uploadFile.pageHeader = Upload supporting documentation
```
Comment (jakebooker56) [inline conf/messages.cy:373]: same here

File: conf/messages.cy [hunk 3]
```diff
@@ -928,15 +915,10 @@ moreDocumentation.error.required = Select yes to send us more documentation
 # Upload Supporting Documentation page
 uploadSupportingDocumentation.pageTitle = Upload documentation
 uploadSupportingDocumentation.pageHeader = Upload documentation
```
Comment (jakebooker56) [inline conf/messages.cy:917]: same here

File: conf/messages.cy (hunk 2 above)
Comment (jacemale-hmrc) [reply conf/messages.cy]: done

**Review by jakebooker56 — APPROVED** [review]

**Review by tvaidotas — APPROVED** [review]

**Pull request merged**